tests/tests_integration/test_api/test_files.py

```diff
@@ -20,6 +20,23 @@ def new_file():
     assert COGNITE_CLIENT.files.retrieve(id=res.id) is None
 
 
+@pytest.fixture(scope="class")
+def new_file_with_label():
+    label_external_id = "my_test_label"
```

Comment:
You may want this label ext id to be random, as (in my experience), it conflicts when github experience runs the code on all four versions of python. 
```suggestion
    import uuid
    label_external_id = uuid.uuid4().hex[0:20]
```
and changing the rest of the code to use a retrieved external id (instead of yielding this external id)

Reply:
Good idea. Done!